Python/TESTED--SSmethod2/run_simulation.py

```diff
@@ -27,7 +27,6 @@
 import cPickle as pickle
 import os
 from glob import glob
-import sys
```

Comment:
@evan-magnusson , Are we not using "import sys" in ths file, or did we want to change something in the body of the file?